juju/juju
# Focal default release LTS

**I'm using CI as another way to test**

Start the process of setting the default supported LTS release to
focal.

## QA steps

Notice that the logs now deploy on focal.

```sh
juju bootstrap lxd test --debug
```
